processing-android

```diff
@@ -15,6 +15,11 @@
 
 # Thanks to Ferdinand Kasper for this build script. [fry]
 
+# Target version
+ANDROID_VERSION="17"
+
+# Your sketchbook location
+SKETCHBOOK_LOCATION="/path/to/your/sketchbook"
```

Comment:
ANDROID_VERSION and SKETCHBOOK_LOCATION can be read from Processing's preferences file, which a specific location for each OS (for example ~/Library/Processing/preferences.txt on Mac OS X).